### ISSUE TITLE
Create repository for pdfjs-dist

### ISSUE THREAD

**Issue by vlaraort** [esm-bundle/new-repo-instructions#25]
Hey!

I would like a new pdfjs-dist repository, as the one in bundled-es-modules (https://github.com/bundled-es-modules/pdfjs-dist/) is unmaintained since 2020 and doesn't have all the accessibility goodies.

### Please create an esm-bundle repo for the following package(s):

https://www.npmjs.com/package/pdfjs-dist

### Steps for esm-bundle maintainer

The maintainers of esm-bundle will do the following:

1. Create a repo from this template. Click on "Use Template". The new repo must be under the `esm-bundle` repo https://github.com/esm-bundle/autopublish-template
2. Create branch protection rule for the `main` branch. Code reviews are required by a code owner before merge, and the `build` github workflow must succeed before merge.

### Steps for new contributor

Once the maintainer has created the repository, fork it and follow these steps: https://github.com/esm-bundle/new-repo-instructions#okay-how-do-i-do-it. Once finished, submit a pull request, which will be reviewed by the maintainers.

**Comment by vlaraort** [comment]
@joeldenning hi?
